src/test/java/de/retest/recheck/cli/subcommands/DiffIT.java

```diff
@@ -61,4 +68,23 @@ public void diff_should_print_differences() throws Exception {
 
 		assertThat( systemOutRule.getLog() ).contains( expected );
 	}
+
+	@Test
+	public void checkFilterNames_should_check_the_filter_names() throws Exception {
+		final String filterName = "positioning.filter";
+		final Diff cut = new Diff();
+		final List<Filter> filters = new ArrayList<>();
+		final Optional<Filter> filter = SearchFilterFiles.searchFilterByName( filterName );
+		filters.add( RecheckIgnoreUtil.loadRecheckIgnore() );
+		if ( filter.isPresent() ) {
```

Comment:
Try to minimize `Optional.isPresent` if you can use lambda methods `Optional.ifPresent`